statick_tool/args.py

```diff
@@ -28,10 +28,10 @@ def __init__(self, name):
         self.parser = argparse.ArgumentParser(description=name)
         self.parser.add_argument("--user-paths", **user_path_args)
 
-    def get_user_paths(self):
+    def get_user_paths(self, args=None):
         """Get a list of user paths containing config or plugins."""
         user_paths = []
-        args = self.pre_parser.parse_known_args()[0]
+        args = self.pre_parser.parse_known_args(args)[0]
```

Comment:
This is safe because the default signature of parse_known_args is (args=None, namespace=None), so if args is None, then the behavior is completely unchanged.